src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -0,0 +1,24 @@
+(ns puppetlabs.trapperkeeper.services.authorization.authorization-service
+  (:require [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [puppetlabs.trapperkeeper.services :refer [service-context]]
+            [puppetlabs.trapperkeeper.authorization.ring-middleware :refer
+             [wrap-authorization-check]]
+            [puppetlabs.trapperkeeper.services.authorization.authorization-core :refer :all]))
+
+(defprotocol AuthorizationService
+  (wrap-handler [this handler]))
```

Comment:
What do you think of renaming `wrap-handler` to `wrap-with-authorization-check`?  Looking at the integration test, `(wrap-handler handler)` seems a bit ambiguous.